app/controllers/api/v2/play_roles_controller.rb

```diff
@@ -0,0 +1,93 @@
+module Api
+  module V2
+    # API controller for Ansible Plays
+    class PlayRolesController < ::Api::V2::BaseController
+      include ::Api::Version2
+      include ForemanTasks::Triggers
+
+      rescue_from ActiveRecord::RecordNotFound, :with => :resource_not_found
+
+      before_action :find_hosts, :only => [:play_roles_on_host]
```

Comment:
Hey, first of all, thank you very much for the review! I'm looking forward to learn a lot from the input. :)

Unfortunately, this is the first time I work with Ruby, which is why I would appreciate some more information regarding these changes. I saw that `find_resource` was used in the other controllers, but I was not sure how this mechanism can be applied to my controller as well. Do you mean that I should remove the play_roles_controller and instead add the methods to the API v2 hosts_controller as hosts_controller_extensions (respectively hostgroups_controller and hostgroups_controller_extensions)?